### ISSUE TITLE
New Node:Cloner generater is needed

### ISSUE THREAD

**Issue by kdsin23** [harisreedhar/AN_BLUEFOX#10]
![cloner](https://user-images.githubusercontent.com/64368228/80740941-6b772580-8b4b-11ea-8be8-1858e0c97b6c.png)

Transform mesh node is used to be cloner meshes, i think if we can delvelop a node, it maybe save us a lot of time to create list of nodes.

**Comment by harisreedhar** [comment]
Can u explain what are the functions of this node?

**Comment by harisreedhar** [comment]
Is this offset matrix node and transform mesh combined?

**Comment by kdsin23** [comment]
> Is this offset matrix node and transform mesh combined?

yes,it is. this node is just used to duplicate some meshes,there no more other functions.

**Comment by harisreedhar** [comment]
> > Is this offset matrix node and transform mesh combined?
> 
> yes,it is. this node is just used to duplicate some meshes,there no more other functions.

But why this node? transform mesh node is not that complicated to setup though

![transform](https://user-images.githubusercontent.com/46858047/80791525-8aa79e80-8baf-11ea-87bc-d6be75c20135.PNG)

**Comment by kdsin23** [comment]
> But why this node? transform mesh node is not that complicated to setup though

using a group node to combine a present.

![mesh transform](https://user-images.githubusercontent.com/64368228/80793025-83d95580-8bc8-11ea-8df5-f39c613edcc5.png)
